apisix/plugins/openid-connect.lua

```diff
@@ -504,7 +519,7 @@ function _M.rewrite(plugin_conf, ctx)
         -- provider's authorization endpoint to initiate the Relying Party flow.
         -- This code path also handles when the ID provider then redirects to
         -- the configured redirect URI after successful authentication.
-        response, err, _, session  = openidc.authenticate(conf, nil, unauth_action, conf.session)
+        response, err, _, session  = openidc.authenticate(conf, path, unauth_action, conf.session)
```

Comment:
Why do you change this?

Reply:
It's just for the code to be better understood, it doesn't change the code logic, the default used is also ngx.var.request_uri.   https://github.com/zmartzone/lua-resty-openidc/blob/master/lib/resty/openidc.lua#L1461

Reply:
Do not make irrelevant changes